### ISSUE TITLE
GSLB_Domain remark statement doesn't update Wide-IP description field

### ISSUE THREAD

**Issue by GWilczek** [F5Networks/f5-appsvcs-extension#789]
### Environment
 * Application Services Version: tested on 3.41.0 and 3.48.0
 * BIG-IP Version: 16.1.3.3

### Summary
"remark" key does not update GSLB_Domain description field as it does for other services like Service_TCP or Service_HTTPS...

### Steps To Reproduce
1. Submit the following declaration:
```json
{
  "$schema": "https://raw.githubusercontent.com/F5Networks/f5-appsvcs-extension/master/schema/latest/as3-schema.json",
  "class": "AS3",
  "action": "deploy",
  "declaration": {
    "class": "ADC",
    "schemaVersion": "3.41.0",
    "gslb_wide_ip_01_tenant": {
      "class": "Tenant",
      "gslb_wide_ip_01_app": {
        "class": "Application",
        "gslb_wide_ip_01_local_domain": {
          "class": "GSLB_Domain",
          "remark": "WIDE_IP DESCRIPTION",
          "domainName": "myexample.labs",
          "resourceRecordType": "A",
          "poolLbMode": "ratio",
          "pools": [
            {
              "use": "gslb_wide_ip_01_local_pool"
            }
          ],
          "iRules": [],
          "persistCidrIpv4": 24,
          "persistenceEnabled": false,
          "ttlPersistence": 3600
        },
        "gslb_wide_ip_01_local_pool": {
          "class": "GSLB_Pool",
          "members": [
            {
              "ratio": 10,
              "server": {
                "bigip": "/Common/dc1-bigip1"
              },
              "virtualServer": "/labs_labs_fqdn_sni_tenant/labs_labs_fqdn-sni_443_https_app/serviceMain",
              "enabled": true,
              "dependsOn": "none"
            }
          ],
          "resourceRecordType": "A",
          "enabled": true,
          "lbModePreferred": "topology",
          "lbModeAlternate": "round-robin",
          "lbModeFallback": "return-to-dns",
          "ttl": 30,
          "verifyMemberEnabled": true,
          "maxAnswersReturned": 1
        }
      }
    }
  }
}

```

2. Observe the following error response:

No error, response is correct:

```json
{
    "id": "a6c1956d-39a0-465e-916f-b342a6e10c9e",
    "results": [
        {
            "code": 200,
            "message": "success",
            "lineCount": 18,
            "host": "localhost",
            "tenant": "gslb_wide_ip_01_tenant",
            "runTime": 1458
        }
    ],
    "declaration": {
        "gslb_wide_ip_01_tenant": {
            "class": "Tenant",
            "gslb_wide_ip_01_app": {
                "class": "Application",
                "gslb_wide_ip_01_local_domain": {
                    "class": "GSLB_Domain",
                    "remark": "WIDE_IP DESCRIPTION",
                    "domainName": "myexample.labs",
                    "resourceRecordType": "A",
                    "poolLbMode": "ratio",
                    "pools": [
                        {
                            "use": "gslb_wide_ip_01_local_pool"
                        }
                    ],
                    "iRules": [],
                    "persistCidrIpv4": 24,
                    "persistenceEnabled": false,
                    "ttlPersistence": 3600
                },
                "gslb_wide_ip_01_local_pool": {
                    "class": "GSLB_Pool",
                    "members": [
                        {
                            "ratio": 10,
                            "server": {
                                "bigip": "/Common/dc1-bigip1"
                            },
                            "virtualServer": "/labs_labs_fqdn_sni_tenant/labs_labs_fqdn-sni_443_https_app/serviceMain",
                            "enabled": true,
                            "dependsOn": "none"
                        }
                    ],
                    "resourceRecordType": "A",
                    "enabled": true,
                    "lbModePreferred": "topology",
                    "lbModeAlternate": "round-robin",
                    "lbModeFallback": "return-to-dns",
                    "ttl": 30,
                    "verifyMemberEnabled": true,
                    "maxAnswersReturned": 1
                }
            }
        },
        "class": "ADC",
        "schemaVersion": "3.41.0",
        "controls": {
            "class": "Controls",
            "userAgent": "vscode-f5/3.15.1",
            "archiveTimestamp": "2023-12-18T18:58:11.059Z"
        },
        "id": "autogen_bfcc3bbb-e85f-48f5-9bbc-95f232c6c057",
        "updateMode": "selective"
    }
}
```

### Expected Behavior
```
root@(dc1-gtm01)(cfg-sync In Sync)(Active)(/Common)(tmos)# list gtm wideip a /gslb_wide_ip_01_tenant/gslb_wide_ip_01_app/myexample.labs description
gtm wideip a /gslb_wide_ip_01_tenant/gslb_wide_ip_01_app/myexample.labs {
    description WIDE_IP DESCRIPTION
}
```

### Actual Behavior
```
root@(dc1-gtm01)(cfg-sync In Sync)(Active)(/Common)(tmos)# list gtm wideip a /gslb_wide_ip_01_tenant/gslb_wide_ip_01_app/myexample.labs description
gtm wideip a /gslb_wide_ip_01_tenant/gslb_wide_ip_01_app/myexample.labs {
    description none
}
```

**Comment by RobCupples** [comment]
This has entered our system as AUTOTOOL-4128 but I suspect this has already been fixed for 3.49 (to be released in January) as part of AUTOTOOL-4056 adding support for the NAPTR record resource type in the GSLB_Domain class.

**Comment by sunitharonan** [comment]
This has been fixed in AS3 3.49
